cypress/integration/navbar.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('Navbar test', () => {
+  it('items should be visible', () => {
+    cy.visit('/');
+    cy.viewport(1280, 720);
+    cy.get('.makeStyles-root-2').should('be.visible');
+    cy.get('.makeStyles-logo-5').should('be.visible');
+    cy.get('.MuiToolbar-root > [href="/backpacks"]').should('be.visible');
+    cy.get('.MuiToolbar-root > [href="/backpacks"]').should('be.visible');
```

Comment:
I gess here should be not three same rows but backpacks, bags,accessories

Reply:
Fixed